android/java/org/chromium/chrome/browser/settings/BraveNewsPreferences.java

```diff
@@ -44,10 +52,14 @@
     public static final String PREF_SHOW_OPTIN = "show_optin";
     public static final String PREF_SOURCES_SECTION = "your_sources_section";
     public static final String PREF_ADD_SOURCES = "add_source_news";
+    public static final String PREF_ADD_RSS_SOURCES = "news_source_1";
```

Comment:
It seems this constant does not used

Reply:
> thanks for resolving all the notices @alexsafe

thanks for pointing them @AlexeyBarabash